### PR TITLE
WIP: 国际化翻译 mr

### PR DESCRIPTION
**Pull request by jiayuan929**

### CODE REVIEW

File: webfe/package_vue/src/language/lang/en.js [hunk 1]
```diff
@@ -1,2163 +0,0 @@
...
-  '个环境变量，将增量更新到当前': 'env variables to update increments to the current',
-  '模块暂无环境变量，请选择其它模块': 'There are no env variables for the module, please select another module',
-  权限管理功能目前只支持主模块: 'The permission management function only supports the default module',
-  新增模块: 'Add module',
-  应用目前不允许创建其它模块: 'The APP is currently not allowed to create other modules',
-  当前应用不允许新增模块: 'The current APP does not allow new modules',
-  帮助: 'Help',
-  访问入口: 'Network & Domains',
-  代码检查: 'Code Analysis',
-  联系BK助手: 'Contact BK Assistant',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:124]: 这里有问题，我一会再来给正确的

File: webfe/package_vue/src/language/lang/en.js [hunk 2]
```diff
@@ -1,2163 +0,0 @@
...
-  暂无描述: 'No description yet',
-  '绑定项目源码仓库：': 'Bind the project source code repository: ',
-  接入指引: 'Access guide',
-  '如需接入统计，请在页面尾部的': 'To access the statistics, please find at the end of the page',
-  '标签前，增加以下代码：': 'Before the label, add the following code: ',
-  '注意：所有使用蓝鲸 Python(Django) 框架的应用默认已嵌入该脚本，无需重复添加。': 'Note: All APP using the BlueKing Python (Django) framework have embedded this script by default, no need to add it repeatedly.',
-  把以下代码放在最后: 'Put the following code at the end',
-  '其他注意事项：': 'Other notes: ',
-  '只有通过以下地址访问，才能保证数据被正确统计：': 'Only access through the following addresses can ensure that the data is correctly counted：',
-  '蓝鲸PaaS平台 v3 版本分配的独立域名，如：': 'The independent domain name assigned by the v3 version of the BlueKing PaaS platform, such as:',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:425]: The independent domain name assigned by the v3 version of the BlueKing PaaS, such as:

File: webfe/package_vue/src/language/lang/en.js [hunk 3]
```diff
@@ -1,2163 +0,0 @@
...
-  '蓝鲸PaaS平台分配的子路径访问地址，如：': 'Sub-path access addresses allocated by the BlueWhale PaaS platform, such as:',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:426]: Sub-path access addresses allocated by the BlueKing PaaS, such as:

File: webfe/package_vue/src/language/lang/en.js [hunk 4]
```diff
@@ -1,2163 +0,0 @@
...
-  在蓝鲸PaaS平台上注册过的独立域名: 'An independent domain name registered on the BlueKing PaaS platform',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:427]: An independent domain name registered on the BlueKing PaaS

File: webfe/package_vue/src/language/lang/en.js [hunk 5]
```diff
@@ -1,2163 +0,0 @@
...
-  '退出应用失败：': 'Failed to quit the APP: ',
-  '角色更换成功！': 'Modify role success!',
-  '角色更换失败：': 'Modify role failed: ',
-  '删除成员成功！': 'Delete member successfully!',
-  '删除成员失败：': 'Failed to delete member:',
-  模块管理: 'Modules',
-  模块的基本信息: 'Basic information of the module',
-  模块名称: 'Module name',
-  模块类型: 'Module type',
-  蓝鲸插件: 'bk-plugin',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:785]: 专有名词？ BK plugin ？

File: webfe/package_vue/src/language/lang/en.js [hunk 6]
```diff
@@ -1,2163 +0,0 @@
...
-  地址: 'Address',
-  镜像信息: 'Mirror information',
-  源码信息: 'Source information',
-  增强服务: 'Add-ons',
-  启用未创建: 'Enable not created',
-  已创建实例: 'Instance created',
-  运行时信息: 'APP Runtime',
-  基础镜像: 'Base image',
-  构建工具: 'Build tools',
-  '确认停止【构建阶段】吗？': 'Are you sure to stop【Build Phase】?',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:982]: Are you sure to stop "Build Phase" ?

File: webfe/package_vue/src/language/lang/en.js [hunk 7]
```diff
@@ -1,2163 +0,0 @@
...
-  '确认停止【部署阶段】吗？': 'Are you sure you want to stop the【Deployment Phase】?',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:983]: Are you sure you want to stop the "Deployment Phase" ?

File: webfe/package_vue/src/language/lang/en.js [hunk 8]
```diff
@@ -1,2163 +0,0 @@
...
-  '应用在蓝鲸桌面打开时的窗口大小，建议选择1200px*600px': 'The window size of the APP when the BlueKing desktop is opened, it is recommended to choose 1200px*600px',
-  '拉伸窗口：': 'Stretched window: ',
-  请选择应用联系人: 'Please select an APP contact',
-  应用联系人不能超过4人: 'APP contacts cannot exceed 4 people',
-  允许: 'Allow',
-  不允许: 'Not allowed',
-  请输入应用名称: 'Please enter an APP name',
-  请输入应用简介: 'Please enter an APP brief',
-  '信息保存成功！': 'Information saved successfully!',
-  '应用市场 (移动端)': 'APP Market (Mobile)',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1221]: APP Market  要是作为功能的话， Market 跟 APP 在一起，统一用 APP Market ，不是 APP market

File: webfe/package_vue/src/language/lang/en.js [hunk 9]
```diff
@@ -1,2163 +0,0 @@
...
-  '如何搭建本地开发环境？': 'How do I build a local development environment?',
-  '使用蓝鲸统一开发环境，本地快速搭建': 'Use the BlueKing Unified Development Environment for a quick local build',
-  '如何部署蓝鲸应用？': 'How to deploy a BlueKing APP?',
-  '如何使用蓝鲸开发框架？': 'How to use the BlueKing APP framework?',
-  '集成登录鉴权、安全防护、权限控制等基础模块，更有后台任务、云API调用等样例供您参考': 'Basic modules such as login authentication, security protection and permission control are integrated, and there are also samples of background tasks and cloud API calls for your reference.',
-  普通应用: 'APP',
-  'S-mart 应用': 'S-mart APP',
-  '云原生 应用': 'Cloud Native APP',
-  容器信息: 'Container information',
-  容器端口: 'container port',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1355]: Container port

上下文推测，好像要大写

File: webfe/package_vue/src/language/lang/en.js [hunk 10]
```diff
@@ -1,2163 +0,0 @@
...
-  暂无应用: 'No APP yet',
-  默认: 'Default',
-  旧应用迁移: 'Migrate Legacy APP',
-  '您可以将蓝鲸应用从旧版开发者中心迁移到新版，我们将为您提供更好的服务能力。': 'You can migrate your APP from the old developer center to the new version and we will provide you with better service capabilities.',
-  执行: 'Execute',
-  回滚: 'Rollback',
-  '在当前迁移流程完成前 [执行迁移-部署新应用-访问新应用-测试验证-确认迁移]，你将无法开始新的迁移工作。': 'You will not be able to start a new migration until the current migration process is complete [Perform Migration - Deploy New APP - Access New APP - Test Verification - Confirm Migration].',
-  '接下来，您需要：': 'Next, you need: ',
-  '1. 在新版开发者中心确认应用第三方地址是否正确': '1. In the new version of the developer center, confirm whether the third-party address of the APP is correct',
-  '2. 当你确认一切功能正常运行后，便可以点击【确认迁移】按钮来完成迁移。': '2. After you confirm that all functions are running normally, you can click the [Confirm Migration] button to complete the migration.',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1420]: 2. After you confirm that all functions are running normally, you can click the "Confirm Migration" button to complete the migration.

File: webfe/package_vue/src/language/lang/en.js [hunk 11]
```diff
@@ -1,2163 +0,0 @@
...
-  语言: 'Language',
-  暂无模块: 'No module available',
-  点击创建新模块: 'Click to create a new module',
-  暂无应用: 'No APP yet',
-  默认: 'Default',
-  旧应用迁移: 'Migrate Legacy APP',
-  '您可以将蓝鲸应用从旧版开发者中心迁移到新版，我们将为您提供更好的服务能力。': 'You can migrate your APP from the old developer center to the new version and we will provide you with better service capabilities.',
-  执行: 'Execute',
-  回滚: 'Rollback',
-  '在当前迁移流程完成前 [执行迁移-部署新应用-访问新应用-测试验证-确认迁移]，你将无法开始新的迁移工作。': 'You will not be able to start a new migration until the current migration process is complete [Perform Migration - Deploy New APP - Access New APP - Test Verification - Confirm Migration].',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1417]: You will not be able to start a new migration until the current migration process is complete "Perform Migration - Deploy New APP - Access New APP - Test Verification - Confirm Migration".

File: webfe/package_vue/src/language/lang/en.js [hunk 12]
```diff
@@ -1,2163 +0,0 @@
...
-  迁移文档: 'migration doc',
-  的指引操作: '',
-  '应用没有使用 Stackless Python 的相关特性': 'The APP does not use features related to Stackless Python',
-  '应用没有使用NFS服务（之前的文件上传采用的NFS）': 'The APP does not use NFS services',
-  '应用如果对外提供了接口，需要通知调用方修改接口url': 'The APP needs to notify the caller to modify the interface url if the interface is provided externally',
-  '没有使用api gateway，需要通知所有使用到接口的应用或服务 修改url': 'No api gateway is used, all APP or services that use the interface need to be notified to modify the url',
-  '使用了api gateway, 只需要再api gateway 上修改url即可': 'api gateway is used, just change the url on the api gateway',
-  '发给其他人或第三方系统的回调链接需要修改，比如邮件中的链接，权限审批链接等': 'Callback links to other people or third party systems need to be modified, e.g. links in emails, permission approval links, etc.',
-  已阅读: 'Read',
-  '点击【开启迁移】后，你的应用服务将不会受到任何影响': 'Your APP services will not be affected in any way after clicking [Enable Migration]',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1465]: Your APP services will not be affected in any way after clicking "Enable Migration"

File: webfe/package_vue/src/language/lang/en.js [hunk 13]
```diff
@@ -1,2163 +0,0 @@
...
-  多语言: 'Multi-language',
-  '支持包含 Python、Go、Node.js 在内的多种编程语言': 'Support for multiple programming languages including Python, Go, Node.js.',
-  弹性扩缩容: 'Elastic scaling',
-  '应用引擎为蓝鲸应用提供了强大的扩缩容能力。应用开发者无需担心访问量增长带来的容量问题，一切都可以交由应用引擎处理': 'The APP engine provides powerful expansion and contraction capabilities for BlueKing . APP developers do not need to worry about the capacity problems caused by the increase in traffic, and everything can be handled by the APP engine.',
-  '蓝鲸Model在线生成器服务，基于Django框架，支持在线配置Django Model模型，一键生成对应Python代码': 'BlueKing Model online generator service, based on the Django framework, supports online configuration of Django Model models, and generates corresponding Python code with one click',
-  在线定义Model: 'Define Model Online',
-  '支持在线定义Django Model，并自定义Model字段，支持常用字段类型。': 'Support online definition of Django Model, and customize Model fields, support common field types.',
-  在线生成源代码: 'Generate source code online',
-  '支持在线一键生成Model源代码，您可在线查看、编辑、下载代码，集成到蓝鲸应用中。': 'Support online one-click generation of Model source code, you can view, edit, download the code online, and integrate it into the BlueKing APP.',
-  '帮助文档：': 'Help documentation: ',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1537]: Documentation:

一个单词就够了

File: webfe/package_vue/src/language/lang/en.js [hunk 14]
```diff
@@ -1,2163 +0,0 @@
...
-  'Django Model生成器使用入门': 'Getting Started with the Django Model Generator',
-  '标准运维V3使用的分布式的流程定义、管理、调度引擎。': 'The distributed process definition, management, and scheduling engine used by bksops.',
-  '来自标准运维 v3': 'from bksops',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1540]: from BKSops

File: webfe/package_vue/src/language/lang/en.js [hunk 15]
```diff
@@ -1,2163 +0,0 @@
...
-  '可以解析、管理、执行由用户创建的流程任务，提供了如暂停、继续、撤销、跳过和重试等灵活的任务控制能力，并且支持子流程，流程串行、并行、分支、循环、打回控制等进阶特性。': 'It can parse, manage, and execute process tasks created by users, providing flexible task control capabilities such as pause, resume, undo, skip, and retry, and support sub-processes, process serial, parallel, branch, loop, play Advanced features such as return control.',
-  简易的流程构造方式: 'Simple process structure',
-  '可以使用 Bamboo 提供的 builder 轻松地构造复杂的流程。': 'Complex processes can be easily constructed using the builder provided by Bamboo.',
-  完整的流程控制特性: 'Complete Process Control Features',
-  '支持基本的流程启动、暂停、继续、重试和撤销，以及流程串行、并行、分支、循环、打回控制等进阶特性。': 'Support basic process start, pause, resume, retry and undo, as well as advanced features such as process serial, parallel, branch, loop, and return control.',
-  足够的并发处理效率: 'Sufficient concurrent processing efficiency',
-  '基于 Celery，通过将不同性质的任务划分到不同的队列， Bamboo 拥有足够的执行效率，并且可通过水平扩展来进一步提升并发处理能力。': 'Based on Celery, by dividing tasks of different natures into different queues, Bamboo has sufficient execution efficiency, and can further improve concurrent processing capabilities through horizontal expansion.',
-  '帮助文档 ：': 'Help documentation: ',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1548]: Documentation:

File: webfe/package_vue/src/language/lang/en.js [hunk 16]
```diff
@@ -1,2163 +0,0 @@
...
-  快速开始: 'quick start',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1549]: Quick start

File: webfe/package_vue/src/language/lang/en.js [hunk 17]
```diff
@@ -1,2163 +0,0 @@
...
-  数据视图: 'data view',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1550]: Data view

File: webfe/package_vue/src/language/lang/en.js [hunk 18]
```diff
@@ -1,2163 +0,0 @@
...
-  '单视图Widget配置，支持线图、饼图等多种图表配置。': 'Single-view Widget configuration, support line graph, pie chart and other chart configurations.',
-  视图面板配置: 'View panel configuration',
-  '支持在线拖拽，自定义布局视图面板。': 'Support online drag and drop, custom layout view panel.',
-  持续集成: 'CI',
-  '蓝鲸提供源码质量检查，在应用部署之后默认启动，并生成检查报告': 'BlueKing provides source code quality inspection, starts by default after APP deployment, and generates inspection reports',
-  '蓝鲸使用蓝盾 CodeCC 进行源码检查，为您的应用代码质量保驾护航。': 'BlueKing uses CodeCC to check the source code to ensure the quality of your APP code.',
-  应用推荐: 'Featured APP',
-  '蓝鲸官方应用市场提供“推荐应用”广告位，帮助您更好的宣传应用': 'The official APP market of BlueKing provides "Recommended APP" advertising space to help you better promote the APP',
-  可视化开发: 'Lesscode',
-  '蓝鲸智云可视化开发平台提供了前端页面在线可视化拖拽组装、配置编辑、源码生成、二次开发等能力。旨在帮助用户通过尽量少的手写代码的方式快速设计和开发 SaaS': 'The BlueKing Lesscode provides online visual assembly of front-end pages, configuration editing, source code generation, customization development and other capabilities.',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1561]: LessCode

File: webfe/package_vue/src/language/lang/en.js [hunk 19]
```diff
@@ -1,2163 +0,0 @@
...
-  数据视图: 'data view',
-  蓝鲸数据可视化服务: 'BlueKing Data Visualization Service',
-  '单视图Widget配置，支持线图、饼图等多种图表配置。': 'Single-view Widget configuration, support line graph, pie chart and other chart configurations.',
-  视图面板配置: 'View panel configuration',
-  '支持在线拖拽，自定义布局视图面板。': 'Support online drag and drop, custom layout view panel.',
-  持续集成: 'CI',
-  '蓝鲸提供源码质量检查，在应用部署之后默认启动，并生成检查报告': 'BlueKing provides source code quality inspection, starts by default after APP deployment, and generates inspection reports',
-  '蓝鲸使用蓝盾 CodeCC 进行源码检查，为您的应用代码质量保驾护航。': 'BlueKing uses CodeCC to check the source code to ensure the quality of your APP code.',
-  应用推荐: 'Featured APP',
-  '蓝鲸官方应用市场提供“推荐应用”广告位，帮助您更好的宣传应用': 'The official APP market of BlueKing provides "Recommended APP" advertising space to help you better promote the APP',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1559]: APP Market

File: webfe/package_vue/src/language/lang/en.js [hunk 20]
```diff
@@ -1,2163 +0,0 @@
...
-  '个变量，忽略': 'Variables, ignoring',
-  点击前往: 'Click to go',
-  由于当前应用主模块: 'Since the current APP default module',
-  去编辑: 'To edit',
-  '名称：': 'Name: ',
-  访问蓝鲸应用: 'Access the BlueKing APP',
-  '停用后，在': 'After deactivation, in',
-  测试企业号中将不能访问该应用: 'The APP will not be accessible in the test enterprise number',
-  企业号中将不能访问该应用: 'The APP will not be accessible from the enterprise account',
-  '是否停用【': 'Whether to disable【',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1752]: Whether to disable "

File: webfe/package_vue/src/language/lang/en.js [hunk 21]
```diff
@@ -1,2163 +0,0 @@
...
-  '】移动端配置': '】Mobile end configuration',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1753]: " Mobile end configuration

File: webfe/package_vue/src/language/lang/en.js [hunk 22]
```diff
@@ -1,2163 +0,0 @@
...
-  CPU使用率: 'CPU usage',
-  '1小时': '1 hour',
-  '24小时': '24 hours',
-  '7天': '7 days',
-  您可将应用发布到: 'You can publish your APP to',
-  '满足以下迁移前置条件，如果不满足，请联系': 'Meet the following migration prerequisites, if not, please contact',
-  未完成迁移应用: 'Incomplete APP',
-  已迁移应用: 'Migrated APP',
-  暂不支持迁移应用: 'Unsupported APP',
-  '开始前，请确认应用【': 'Before starting, please confirm the APP【',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1771]: Before starting, please confirm the APP "

File: webfe/package_vue/src/language/lang/en.js [hunk 23]
```diff
@@ -1,2163 +0,0 @@
...
-  '】，我们将单独处理您的迁移请求。': '】, we will handle your migration request separately.',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1772]: " , we will handle your migration request separately.

File: webfe/package_vue/src/language/lang/en.js [hunk 24]
```diff
@@ -1,2163 +0,0 @@
...
-  应用预发布环境未下架: 'APP stag env not archived',
-  '回滚失败，请联系': 'Rollback failed, please contact',
-  '应用 [': 'APP [',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1775]: APP "

File: webfe/package_vue/src/language/lang/en.js [hunk 25]
```diff
@@ -1,2163 +0,0 @@
...
-  '] 正在进行': '] ',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1776]: "

File: webfe/package_vue/src/language/lang/en.js [hunk 26]
```diff
@@ -1,2163 +0,0 @@
...
-  蓝鲸提供的监控类服务集合: 'A collection of monitoring services provided by BlueKing',
-  流程引擎: 'Process Engine',
-  '标准运维V3使用的分布式的流程定义、管理、调度引擎': 'The distributed process definition, management and scheduling engine used by bksops.',
-  推广: 'Marketing',
-  '蓝鲸提供的官方应用市场，用户可以在市场中搜索并使用您开发的应用': 'The official APP marke provided by BlueKing, where users can search for and use APP.',
-  蓝鲸官方应用市场提供的推荐宣传广告位: 'Recommended promotional advertising space provided by the official BlueKing APP market',
-  文档: 'Documents',
-  资源与工具: 'Resources',
-  'SDK 文档': 'SDK Docs',
-  社区: 'Community',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1805]: 一会看看nek还要修订不

File: webfe/package_vue/src/language/lang/en.js [hunk 27]
```diff
@@ -1,2163 +0,0 @@
...
-  权限管理功能目前只支持主模块: 'The permission management function only supports the default module',
-  新增模块: 'Add module',
-  应用目前不允许创建其它模块: 'The APP is currently not allowed to create other modules',
-  当前应用不允许新增模块: 'The current APP does not allow new modules',
-  帮助: 'Help',
-  访问入口: 'Network & Domains',
-  代码检查: 'Code Analysis',
-  联系BK助手: 'Contact BK Assistant',
-  蓝鲸桌面: 'BlueKing desktop',
-  技术支持: 'Technical Support',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:126]: Support

File: webfe/package_vue/src/language/lang/en.js [hunk 28]
```diff
@@ -1,2163 +0,0 @@
...
-  社区论坛: 'Community',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:127]: Forum

File: webfe/package_vue/src/language/lang/en.js [hunk 29]
```diff
@@ -1,2163 +0,0 @@
...
-  产品官网: 'Official Website',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:128]: Official

File: webfe/package_vue/src/language/lang/en.js (hunk 26 above)
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1805]: Forum

File: webfe/package_vue/src/language/lang/en.js [hunk 30]
```diff
@@ -1,2163 +0,0 @@
...
-  计算: 'Computing',
-  '提供弹性、便捷的应用部署服务，支持Python、Go 等多种语言': 'Provide flexible and convenient APP deployment services, supporting Python, Go and other languages.',
-  蓝鲸提供的数据存储类服务集合: 'A collection of services provided by BlueKing in the category of data storage.',
-  蓝鲸提供的监控类服务集合: 'A collection of monitoring services provided by BlueKing',
-  流程引擎: 'Process Engine',
-  '标准运维V3使用的分布式的流程定义、管理、调度引擎': 'The distributed process definition, management and scheduling engine used by bksops.',
-  推广: 'Marketing',
-  '蓝鲸提供的官方应用市场，用户可以在市场中搜索并使用您开发的应用': 'The official APP marke provided by BlueKing, where users can search for and use APP.',
-  蓝鲸官方应用市场提供的推荐宣传广告位: 'Recommended promotional advertising space provided by the official BlueKing APP market',
-  文档: 'Documents',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1802]: Documentation

File: webfe/package_vue/src/language/lang/en.js [hunk 31]
```diff
@@ -1,2163 +0,0 @@
...
-  资源与工具: 'Resources',
-  'SDK 文档': 'SDK Docs',
-  社区: 'Community',
-  蓝鲸论坛: 'BlueKing Forum',
```
Comment (HeraZhang) [inline webfe/package_vue/src/language/lang/en.js:1806]: Forum

**Review by HeraZhang — CHANGES_REQUESTED** [review]

File: webfe/package_vue/src/language/lang/en.js (hunk 20 above)
Comment (jiayuan929) [reply webfe/package_vue/src/language/lang/en.js:1752]: 这个是需要根据上下午语境带上的

File: webfe/package_vue/src/language/lang/en.js (hunk 21 above)
Comment (jiayuan929) [reply webfe/package_vue/src/language/lang/en.js:1753]: 这个是需要根据上下午语境带上的

File: webfe/package_vue/src/language/lang/en.js (hunk 22 above)
Comment (jiayuan929) [reply webfe/package_vue/src/language/lang/en.js:1771]: 这个是需要根据上下午语境带上的

File: webfe/package_vue/src/language/lang/en.js (hunk 1 above)
Comment (huangpeng0428) [reply webfe/package_vue/src/language/lang/en.js:124]: Contact BK Customer Service

Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:124]: 已修复

File: webfe/package_vue/src/language/lang/en.js [hunk 32]
```diff
@@ -1,2163 +0,0 @@
...
-  '模块暂无环境变量，请选择其它模块': 'There are no env variables for the module, please select another module',
-  权限管理功能目前只支持主模块: 'The permission management function only supports the default module',
-  新增模块: 'Add module',
-  应用目前不允许创建其它模块: 'The APP is currently not allowed to create other modules',
-  当前应用不允许新增模块: 'The current APP does not allow new modules',
-  帮助: 'Help',
-  访问入口: 'Network & Domains',
-  代码检查: 'Code Analysis',
-  联系BK助手: 'Contact BK Assistant',
-  蓝鲸桌面: 'BlueKing desktop',
```
[the remark this answers is not in the document]
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:125]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 2 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:425]: 已修改

File: webfe/package_vue/src/language/lang/en.js [hunk 33]
```diff
@@ -1,2163 +0,0 @@
...
-  退出: 'Logout',
-  退出登录: 'Sign out',
-  蓝鲸应用: 'BlueKing APP',
-  立即体验: 'Try it now',
-  查看文档: 'View documentation',
-  '搜索中...': 'Searching...',
-  无应用: 'No APP',
-  无相关文档: 'No related documentation',
-  代码仓库: 'Repository',
-  请选择: 'please choose',
```
[the remark this answers is not in the document]
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:144]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 3 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:426]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 4 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:427]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 28 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:127]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 29 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:128]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 26 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:1805]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 30 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:1802]: 已修改

File: webfe/package_vue/src/language/lang/en.js (hunk 31 above)
Comment (leafage-collb) [reply webfe/package_vue/src/language/lang/en.js:1806]: 已修改

**Pull request closed without merging**